app/assets/javascripts/modules/remove-filter.js

```diff
@@ -5,8 +5,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
   'use strict'
 
   GOVUK.Modules.RemoveFilter = function RemoveFilter () {
-    var onChangeSuppressAnalytics = {
-      type: 'change',
+    var suppressAnalytics = {
       suppressAnalytics: true
     }
```

Comment:
Minor, I don't think it's necessary to have a variable of `suppressAnalytics` that equals `{ suppressAnalytics: true }` I think we should just pass the object literal in as an argument to triggerEvent.

Reply:
So: `window.GOVUK.triggerEvent($input.val(newVal)[0], 'change', suppressAnalytics: true)` ?

Reply:
It'll be `window.GOVUK.triggerEvent($input.val(newVal)[0], 'change', { detail: { suppressAnalytics: true } })` (as per: https://github.com/alphagov/finder-frontend/pull/2532#discussion_r649807087)

You also can't omit the curly braces like Ruby lets you.

Reply:
Sorted :)